include/filters.php

```diff
@@ -361,7 +361,14 @@ public function translate_user_email( $email ) {
 	 * @return int
 	 */
 	public function translate_page_for_privacy_policy( $id ) {
-		return empty( $this->curlang ) ? $id : $this->model->post->get( $id, $this->curlang );
+		static $once = false; // Avoid infinite loop.
+
+		if ( ! $once && ! empty( $this->curlang ) ) {
+			$once = true;
+			return $this->model->post->get( $id, $this->curlang );
+		}
+
+		return $id;
```

Comment:
Why is this necessary? I understand the modification of the second method as it deals with caps, but why this one?
Moreover, I fear that this introduces a bug as I expect the result to be wrong at the second call.

Reply:
With the snippet I proposed, the two modification are necessary otherwise there is always the infinite loop.
As you pointed out, I think we could introduce some bugs.
Indeed when working out on the test, I didn't succeed to develop a test as close as the real execution i.e. with a privacy policy page saved as an option precisely because `update_option()` is called in the same process of `get_option()` in the test. 
So I have no other idea to prevent this infinite loop.